sklearn/feature_extraction/text.py

```diff
@@ -253,16 +313,18 @@ def build_preprocessor(self):
                              self.strip_accents)
 
         if self.lowercase:
```

Comment:
We can remove the `if self.lowercase` and have:

```py
return partial(_preprocess, 
               accent_function=strip_accents, 
               lower=self.lowercase)
```

Reply:
Done!